firebase-sessions/src/main/kotlin/com/google/firebase/sessions/settings/RemoteSettings.kt

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.firebase.sessions.settings
+
+import android.content.Context
+import android.net.Uri
+import androidx.datastore.preferences.preferencesDataStore
+import java.net.URL
+import kotlin.time.Duration
+import kotlin.time.Duration.Companion.seconds
+
+internal class RemoteSettings(val context: Context) : SettingsProvider {
+  private val SESSION_CONFIGS_NAME = "firebase_session_settings"
```

Comment:
Nit: Constants like this should be in a companion object and have the `const` modifier.  That will be like a `static final` in Java.

Reply:
Sounds good. Done!